### ISSUE TITLE
Informative Path Planning to Estimate Quantiles for Environmental Analysis. (arXiv:2201.10633v3 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#19602]
Informative Path Planning to Estimate Quantiles for Environmental Analysis. (arXiv:2201.10633v3 [cs.RO] UPDATED)<br>
https://ift.tt/Y13ClJj<br>
<p>Scientists interested in studying natural phenomena often take physical specimens from locations in the environment for later analysis. These analysis locations are typically specified by expert heuristics. Instead, we propose to choose locations for scientific analysis by using a robot to perform an informative path planning survey. The survey results in a list of locations that correspond to the quantile values of the phenomenon of interest. We develop a robot planner using novel objective functions to improve the estimates of the quantile values over time and an approach to find locations which correspond to the quantile values. We test our approach in four different environments using previously collected aquatic data and validate it in a field trial. Our proposed approach to estimate quantiles has a 10.2% mean reduction in median error when compared to a baseline approach which attempts to maximize spatial coverage. Additionally, when localizing these values in the environment, we see a 15.7% mean reduction in median error when using cross-entropy with our loss function compared to a baseline.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
